std/encoding/mod.ts

```diff
@@ -10,5 +10,6 @@ export {
   encode as encodeToHex,
   encodeToString as encodeToHexString
 } from "./hex.ts";
+export { parse as parseMarkdown, MarkdownOptions } from "./markdown.ts";
```

Comment:
`parse as renderMarkdown`?

Reply:
It is named as `parse` in the original library, all the other APIs are exposed as `parseXxx`, I don't see the logic personally in breaking that pattern.  Thoughts from @ry or @bartlomieju?

Reply:
I don't see value in having `std/encoding/mod.ts` - importing this file will import **a lot** of dependencies... That said, if we have it, let's keep it consistent to other already reexported modules.

Reply:
I agree